tools/quasitools/aacoverage.xml

```diff
@@ -1,13 +1,15 @@
-<tool id="aacoverage" name="Amino Acid Coverage" version="0.7.0">
+<tool id="aacoverage" name="Amino Acid Coverage" version="@TOOL_VERSION@">
     <description>Builds an aa census and returns its coverage</description>
-    <requirements>
-          <requirement type="package" version="0.7.0">quasitools</requirement>
-    </requirements>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
     <command detect_errors="exit_code"><![CDATA[
 
-        ln -f -s ${input_bam.metadata.bam_index} ${input_bam}.bai &&
+        ln -s $input_bam '${input_bam.name}' &&
+        ln -s $input_bam.metadata.bam_index '${input_bam.name}.bai' &&
 
-        quasitools aacoverage $input_bam $ref_file $input_genes -o output.csv
+        quasitools aacoverage ${input_bam.name} $ref_file $input_genes -o output.csv
```

Comment:
It's also recommended to add quotes around file names so that e.g., spaces in file names on the command-line won't mess with things. So this could be changed to `'${input_bam.name}'`

Reply:
Ah, good catch. I guess the `.name` part can introduce the spaces, whereas the other parameters will be all `.dat`.